olewoo_cs/Typelib/OWCustData.cs

```diff
@@ -43,5 +43,27 @@ public static void GetCustData(ITypeInfo ti, ref List<string> lprops)
             NativeMethods.ClearCustData(ptr);
             Marshal.FreeHGlobal(ptr);
         }
+
+        public static void GetAllFuncCustData(int index, ITypeInfo ti, ref List<string> lprops)
+        {
+            if (!(ti is ITypeInfo2 t2))
+            {
+                return;
+            }
+
+            var custdata = new CUSTDATA();
+            var ptr = Marshal.AllocHGlobal(Marshal.SizeOf(custdata));
+            t2.GetAllFuncCustData(index, ptr);
```

Comment:
As discussed in mail - this currently causes an exception - pls patch in your other changes?
With the separate patch you sent me, I get an exception rendering scrrun.dll